### ISSUE TITLE
endpoint to delete connector is failing

### ISSUE THREAD

**Issue by mdrillin** [debezium/debezium-ui-poc#214]
Attempt to delete a connector via the rest endpoint is failing with the following message:
"RESTEASY003210: Could not find resource for full path: http://localhost:8080/api/connectors/1/test"

Have verified that the connector does exist.  Also verified with @indraraj that he is getting the same thing.  @rk3rn3r - can you check into this one?

**Comment by rk3rn3r** [comment]
Hey @mdrillin @indraraj. This is not an exception from the code, but from the HTTP REST framework. Looks like the URL is wrong, also what you pasted here it looks correct. Are you sure you were using the DELETE HTTP verb or maybe you have an unprintable char hidden somewhere (this happens sometimes on Apple machines) that gets appended to the URI?

**Comment by mdrillin** [comment]
@rk3rn3r  I double checked the url from UI side to verify something was not miss-typed.  Also tried copy-paste the curl cmd from the swagger page.  There aren't any special chars either, so I'm not sure what is wrong here.

**Comment by rk3rn3r** [comment]
and the HTTP verb is DELETE?

**Comment by rk3rn3r** [comment]
@mdrillin Can you paste the exact curl call here?

**Comment by mdrillin** [comment]
Sure - I'm running the following in succession.  The first successfully gets the connector list, showing that connector 'test' is present.  The second 'DELETE' I get a 404 with the RESTEASY003210 exception

1) curl -X GET "http://localhost:8080/api/connectors/1" -H  "accept: application/json"

2) curl -X DELETE "http://localhost:8080/api/connectors/1/test" -H  "accept: application/json"

**Comment by rk3rn3r** [comment]
@mdrillin can you try adding Content-Type header? like: `curl -X DELETE -H "accept: "application/json" -H "Content-Type:application/json" "http://localhost:8080/api/connectors/1/test"`

**Comment by mdrillin** [comment]
@rk3rn3r I get the same error with that.  The delete is working for @uidoyen , so maybe my backend is out-of-date or something - I'll retry

**Comment by mdrillin** [comment]
Still no luck...  The DELETE still fails with a 404.  Below are screenshots 

Request
![404Error1](https://user-images.githubusercontent.com/1820403/102243620-fb8e9300-3ec0-11eb-8deb-a8068d9a2b4d.png)

Response
![404Error2](https://user-images.githubusercontent.com/1820403/102243658-0812eb80-3ec1-11eb-8f49-428898236111.png)

**Comment by rk3rn3r** [comment]
The last deployed version for the UX testing had a running version which we used for the UX demo. We also have integration tests that confirm the endpoint is working.
Without more information on what version you are running it's very hard to figure out why it's not working for you. We added the delete endpoint late, so maybe the version you are running does not have that endpoint. When you are running your local environment you might have to rebuild the backend.
